src/destination/tpl/selector.tpl

```diff
@@ -4,9 +4,18 @@
     {{/if}}
     <div>
         {{#if description}}
-        <p>{{description}}</p>
+        <p class="section-title">{{description}}</p>
         {{/if}}
         <div class="selector-container"></div>
+        {{#if showACL}}
+        <div class="permissions-settings-container">
+            <p class="section-title">{{__ 'Permission settings'}}</p>
+            <input type="radio" id="acl-keep-original" {{#equal aclTransferMode 'acl.keep.original' }}checked {{/equal}}name="acl-mode" value="acl.keep.original" />
+            <label for="acl-keep-original">{{__ 'Keep the original permissions'}}</label><br>
+            <input type="radio" id="acl-use-destination" {{#equal aclTransferMode 'acl.use.destination' }}checked {{/equal}}name="acl-mode" value="acl.use.destination" />
+            <label for="acl-use-destination">{{__ 'Inherit the permission of the parent folder'}}</label>
+        </div>
```

Comment:
**issue:** There are severals issues in this code:
- The component does not uses a form despite it manages input fields. While this is ok since the component manage the inputs by itself, it may impact the container if it is also a form as the name fields will be part of it silently. Suggestion: don't use `name`, prefer `data-name` or similar.
- The labels are identified, but the ID is not guaranteed to be unique within the page, leading to a possible collision. Suggestion: don't use ID, prefer nested label instead `<label><input /></label>`, or use dynamic unique ID if the nested label cannot apply

Reply:
updated. Unfortunately, removing name from the inputs will remove the relation between the button set. however,  thats something we may fix with some javascript :)

Reply:
Correct, the name allows grouping the buttons. Please ignore my previous comment, the name should be present.